plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/cookiecutter.test.ts

```diff
@@ -65,122 +70,147 @@ describe('fetch:cookiecutter', () => {
 
   const action = createFetchCookiecutterAction({
     integrations,
-    templaters,
+    containerRunner,
     reader: mockReader,
   });
 
-  templaters.register('cookiecutter', cookiecutterTemplater);
-
   beforeEach(() => {
-    mock({ [`${mockContext.workspacePath}/result`]: {} });
-    jest.restoreAllMocks();
+    jest.resetAllMocks();
+
+    mockContext = {
+      input: {
+        url: 'https://google.com/cookie/cutter',
+        targetPath: 'something',
+        values: {
+          help: 'me',
+        },
+      },
+      baseUrl: 'somebase',
+      workspacePath: mockTmpDir,
+      logger: getVoidLogger(),
+      logStream: new PassThrough(),
+      output: jest.fn(),
+      createTemporaryDirectory: jest.fn().mockResolvedValue(mockTmpDir),
+    };
+
+    // mock the temp directory
+    mock({ [mockTmpDir]: {} });
```

Comment:
dude we really should name this import not just `mock` :) `mockFs` seems good to me

Reply:
Ha this was left over from the previous tests.. good call.